--- a/src/main/java/dev/bleepo/customhitdelay/listeners/OnDamage.java
+++ b/src/main/java/dev/bleepo/customhitdelay/listeners/OnDamage.java
@@ -18,9 +18,6 @@ public class OnDamage implements Listener, ConfigManager {
             if (e.getEntity() instanceof Monster || e.getEntity() instanceof Animals) {
                 player.setMaximumNoDamageTicks(plugin.getConfig().getInt("default-ticks"));
             }
-        } else {
-            Player player = (Player) e.getDamager();
-            player.setMaximumNoDamageTicks(plugin.getConfig().getInt("default-ticks"));
         }
     }
 }
